src/mono/mono/mini/aot-compiler.c

```diff
@@ -13962,19 +13962,22 @@ mono_compile_assembly (MonoAssembly *ass, guint32 opts, const char *aot_options,
 			g_ptr_array_add (acfg->method_order,GUINT_TO_POINTER (method_index));
 	}
 
-	acfg->num_trampolines [MONO_AOT_TRAMP_SPECIFIC] = mono_aot_mode_is_full (&acfg->aot_opts) ? acfg->aot_opts.ntrampolines : 0;
+	if (mono_aot_mode_is_interp (&acfg->aot_opts) || mono_aot_mode_is_full (&acfg->aot_opts)) {
```

Comment:
wish this could be checking `MonoEEFeatures:use_aot_trampolines`, but I think that struct is only setup properly for the aot-runtime, not the aot-compiler

Reply:
Yes.